constellations/gemini-admin/src/emailProgram/emailPrograms.js

```diff
@@ -0,0 +1,230 @@
+"use strict";
+require("./emailPrograms.sass");
+import axios from "axios";
+import React from "react";
+import API, { executeApiCalls } from "../api.js";
+import { getFullName } from "../utils/userUtils.js";
+
+export class EmailProgramsPage extends React.Component {
```

Comment:
// Can you add TODO comments at the top here?
// I think for the user section, we need a "Select All" and "Deselect All" buttons since there can potentially be a couple of classes and MANY users.